app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+#!/usr/bin/env python
+
+'''
+Parallel execution of RTC Workflow
+'''
+
+import argparse
+from itertools import repeat
+import logging
+import multiprocessing
+import os
+import subprocess
+import time
+import yaml
+
+import isce3
+import numpy as np
+from osgeo import gdal
+
+from rtc.runconfig import RunConfig
+import rtc_s1
+
+logger = logging.getLogger('rtc_s1')
+
+def get_rtc_s1_parser():
+    '''Initialize YamlArgparse class and parse CLI arguments for OPERA RTC.
+    Modified after copied from `rtc_s1.py`
```

Comment:
```suggestion
```

Reply:
Thanks for catching this. Please note that I've modified the code to import `get_rtc_s1_parser` from `rtc_s1.py`, and make use of that for get the argument parser (See below). Therefore, `get_rtc_s1_parser()` in `rtc_s1_parallel.py` will be removed in this iteration.

```python
# in rtc_s1_parellel.py
def main():
    '''
    Main entrypoint of the script
    '''
    parser  = rtc_s1.get_rtc_s1_parser()
    # parse arguments
    args = parser.parse_args()

```

Reply:
Sounds good. We can remove the sentence: `Modified after copied from `rtc_s1.py` because other developers don't need to know that that function was copied from `rtc_s1.py`.

Reply:
Sorry, @seongsujeong . I missed that you are removing `get_rtc_s1_parser()` from you module. Don't you need to provide the user with an CLI option to setup the number of workers? Or you are removing it so you can determine the number using the runconfig?